SuperBuild/External_ITK.cmake

```diff
@@ -27,13 +27,15 @@ if(NOT DEFINED ITK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_REPOSITORY
-    "${EP_GIT_PROTOCOL}://github.com/Slicer/ITK"
+    # "${EP_GIT_PROTOCOL}://github.com/Slicer/ITK"
+    "${EP_GIT_PROTOCOL}://github.com/Kitware/ITK"
```

Comment:
https://github.com/InsightSoftwareConsortium/ITK is the source of truth. `github.com/Kitware/ITK` is one of the mirrors.

Reply:
Makes sense, yes, thanks.

@dzenanz do you know if Slicer/ITK had any remaining patches compared to ITKv5?

Reply:
I don't think so. @phcerdan, @sjh26 or @jcfr might know that with more certainty.